### ISSUE TITLE
使用paddlex的基于Docker的TensorRT部署paddleseg模型报错

### ISSUE THREAD

**Issue by lpt-hub** [PaddlePaddle/PaddleSeg#1478]
环境：win10,cuda11,paddleseg2.3,paddlex2.0,bisenetv2

模型导出命令：python export.py --config configs\quick_start/bisenet_optic_disc_512x512_1k-100000.yml --model_path output\best_model\model.pdparams --input_shape 1 3 1024 1024 --save_dir output

转换onnx命令：paddle2onnx --model_dir output --model_filename output/model.pdmodel --params_filename output/model.pdiparams --opset_version 11 --save_file output.onnx

配置文件：
batch_size: 4
iters: 100000

train_dataset:
type: Dataset
dataset_root: dataset\bedwetcrack-train
train_path: dataset\bedwetcrack-train\train_list.txt
num_classes: 3
transforms:
- type: Resize
target_size: [1024, 1024]
- type: RandomHorizontalFlip
- type: Normalize
mode: train

val_dataset:
type: Dataset
dataset_root: dataset\bedwetcrack-train
val_path: dataset\bedwetcrack-train\val_list.txt
num_classes: 3
transforms:
- type: Resize
target_size: [1024, 1024]
- type: Normalize
mode: val

input:

name: "x"
data_type: TYPE_FP32
dims:
1
3
1024
1024
output:
name: "save_infer_model/scale_0.tmp_1"
data_type: TYPE_FP32
dims:
1
3
1024
1024
optimizer:
type: sgd
momentum: 0.9
weight_decay: 4.0e-5

lr_scheduler:
type: PolynomialDecay
learning_rate: 0.01
power: 0.9
end_lr: 0

loss:
types:
- type: CrossEntropyLoss
- type: CrossEntropyLoss
- type: CrossEntropyLoss
- type: DiceLoss
- type: DiceLoss
coef: [1, 1, 1, 1, 1]

model:
type: BiSeNetV2
pretrained: Null

使用paddleseg导出模型的。
模型输入输出大小：
![image](https://user-images.githubusercontent.com/63625557/139204708-b52a03b6-b0c4-45c9-8e55-2353f6db9738.png)

推理命令：./build/demo/model_infer --image files/123.jpg --cfg_file files/bisenet.yml --model_type seg --model_file files/output.onnx

报错情况：
ParseCommandLineFlags:FLAGS_model_type=seg model_file=files/output.onnx
REGISTER_CLASS:seg
init SegModel,model_type=seg
start model init
start engine init
Input filename: files/output.onnx
ONNX IR version: 0.0.7
Opset version: 11
Producer name: PaddlePaddle
Producer version:
Domain:
Model version: 0
Doc string:
WARNING: Logging before InitGoogleLogging() is written to STDERR
W1027 09:25:13.052062 5311 tensorrt_engine.h:60] /home/jenkins/workspace/OSS/L0_MergeRequest/oss/parsers/onnx/onnx2trt_utils.cpp:220: Your ONNX model has been generated with INT64 weights, while TensorRT does not natively support INT64. Attempting to cast down to INT32.
W1027 09:25:13.058593 5311 tensorrt_engine.h:60] Tensor DataType is determined at build time for tensors not marked as input or output.
I1027 09:25:15.795220 5311 tensorrt_engine.h:57] Some tactics do not have sufficient workspace memory to run. Increasing workspace size may increase performance, please check verbose output.
I1027 09:25:32.534358 5311 tensorrt_engine.h:57] Detected 1 inputs and 1 output network tensors.
start model predict 1
------------1
Segmentation fault (core dumped)

**Comment by juncaipeng** [comment]
在paddlex有相同issue，请关注那个issue。
